tests/sparseml/pytorch/optim/test_modifier_quantization.py

```diff
@@ -88,6 +92,7 @@ def _test_quantizable_module(module, qat_expected, modifier):
         )
```

Comment:
let's add a test here for `if modifier.activation_bits is not None` and check that the qconfig activation range is correct

Reply:
Done!